Embedded/MaxMix/Display.ino

```diff
@@ -88,23 +89,30 @@ void DisplayGameSelectScreen(Adafruit_SSD1306* display, char* name, uint8_t volu
 //---------------------------------------------------------
 // Draws the Application mode Edit screen
 //---------------------------------------------------------
-void DisplayApplicationEditScreen(Adafruit_SSD1306* display, char* name, uint8_t volume, bool isMuted, uint8_t modeIndex, uint8_t modeCount)
+void DisplayApplicationEditScreen(Adafruit_SSD1306* display, char* name, uint8_t volume, bool isMuted, uint8_t modeIndex, uint8_t modeCount, uint32_t* displayScrollTimer, uint32_t now)
 {
   display->clearDisplay();
```

Comment:
calculate the new scrollOffset before clearing the display. If the scroll offset has not changed from last call, return. If it has, store the results and continue.

Reply:
Note: This should be done in the other locations as well where you are scrolling.